src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PackageRestore/IVsProjectRestoreInfoSource.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
```

Comment:
IVsProjectRestoreInfoSource.cs is just temporary . Will delete this file. Please ignore it @lifengl